src/activity-logger/background/log-page-visit.js

```diff
@@ -52,17 +58,23 @@ export async function logPageVisit({ tabId, url }) {
     // The time to put in documents.
     const timestamp = Date.now()
 
-    // First create an identifier for the page being visited.
-    const storePageResult = await storePage({ tabId, url })
+    const pageId = generatePageDocId({ url })
+    const existingPage = index.initSingleLookup()(pageId)
```

Comment:
This needs an `await`. Now it will throw an error every single time you visit a page (as `existingPage` will be a `Promise` but the code treats it like a page object)